src/libraries/System.Runtime/tests/System/Runtime/JitInfoTests.cs

```diff
@@ -0,0 +1,151 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.XUnitExtensions;
+using System.Threading;
+using Xunit;
+
+namespace System.Runtime.Tests
+{
+    public class JitInfoTests
+    {
+        private const TestPlatforms AotPlatforms = TestPlatforms.Browser | TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.Android;
+
+        [Fact]
```

Comment:
```suggestion
        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsMonoAOT))] // JitInfo metrics will be 0 in AOT scenarios
```

(and delete Fact and SkipOnPlatform attributes.  For some reason GH isn't letting me comment on multiple lines today)